apps/dashboard/src/components/workflow-editor/steps/configure-step.tsx

```diff
@@ -99,4 +100,12 @@ export function ConfigureStep() {
       </motion.div>
     </>
   );
-}
+};
+
+export const ConfigureStep = () => {
+  return (
+    <StepEditorProvider>
+      <ConfigureStepInternal />
+    </StepEditorProvider>
+  );
+};
```

Comment:
Following similar logic as with EditStepSidebar. They're mounted on different URL paths therefore we need to wrap both of them to this provider. 

```suggestion
export const EditStepSidebar = () => {
  return (
    <StepEditorProvider>
      <EditStepSidebarInternal />
    </StepEditorProvider>
  );
};

```